slurmci-test.toml

```diff
@@ -37,5 +40,6 @@ cpu_gpu = [
 
 gpu = [
   { file = "test/DGmethods/advection_diffusion/pseudo1D_advection_diffusion.jl", n = 3, args = ["true"] },
+  { file = "examples/Atmos/heldsuarez.jl", n = 3, args = [] },
```

Comment:
I think this needs:

`   { file = "test/DGmethods/compressible_Navier_Stokes/rayleigh-benard_model.jl", n = 3, args = [] },`

Reply:
ah, good point.